src/qt/forms/optionsdialog.ui

```diff
@@ -118,6 +118,69 @@
         </layout>
        </item>
        <item>
+        <widget class="QGroupBox" name="groupBox_3">
+         <property name="title">
+          <string>Disk usage</string>
+         </property>
+         <layout class="QVBoxLayout" name="verticalLayout_3">
+          <item>
+           <widget class="QCheckBox" name="pruneEnableChk">
+            <property name="text">
+             <string>Limit blockchain size (prune)</string>
+            </property>
+           </widget>
+          </item>
+          <item>
+           <layout class="QHBoxLayout" name="horizontalLayout">
+            <item>
+             <widget class="QLabel" name="label">
+              <property name="text">
+               <string>Use at most</string>
+              </property>
+             </widget>
+            </item>
+            <item>
+             <widget class="QDoubleSpinBox" name="pruneSize">
+              <property name="suffix">
+               <string> GB</string>
+              </property>
+              <property name="minimum">
+               <double>0.550000000000000</double>
+              </property>
+              <property name="maximum">
+               <double>9999.000000000000000</double>
+              </property>
+              <property name="value">
+               <double>200.000000000000000</double>
+              </property>
+             </widget>
+            </item>
+           </layout>
+          </item>
+          <item>
+           <widget class="QLabel" name="label_2">
+            <property name="text">
+             <string>Note that additional disk space (~2GB) is required for other datasets</string>
```

Comment:
If you cherry-pick https://github.com/bitcoin/bitcoin/commit/b1a8374aaa82e08de8ac9353e76be3cdf43cd3d5, you'd be able to replace the hardcoded value with `CHAIN_STATE_SIZE` (which is the size of the "empty" datadir)